docs/release/release-2.1.rst

```diff
@@ -0,0 +1,8 @@
+===========
+Release 2.1
+===========
+
+Bug Fixes
+---------
+* Fix bug with ``server start`` when only frontend LDAP in Presto is enabled.
+* Add unique temporary directory when issuing ``get`` in parallel.
```

Comment:
Make this user-facing -- "Fix intermittent bug with ``server start`` printing out irrelevant error messages"